### PR TITLE
Feature/dynamic alt profiles concepts

### PR DESCRIPTION
**Pull request by matcor-bodc**

### CODE REVIEW

File: nvsvocprez/app.py [hunk 1]
```diff
@@ -1901,16 +1922,25 @@ def _render_puv_rdf(self):
...
+              FILTER ( ?p != skos:narrower )
+              FILTER ( ?p != skos:related )
+              FILTER ( ?p != owl:sameAs )
+              # exclude other properties from altprof
+              <EXCLUDE_FILTERS>
             }
             """.replace(
             "xxx", self.instance_uri
+        ).replace(
+            "<EXCLUDE_FILTERS>", "\n".join(exclude_filters)
```
Comment (roje-bodc) [inline nvsvocprez/app.py:1937]: You can convert this to an f string and using double braces to escape the any single braces within the query. 

i.e.

```python
"""
WHERE {{ 
              FILTER ( ?p != skos:broaderTransitive )
              FILTER ( ?p != skos:narrowerTransitive )
              FILTER ( ?p != skos:broader )
              FILTER ( ?p != skos:narrower )
              FILTER ( ?p != skos:related )
              FILTER ( ?p != owl:sameAs )
              # exclude other properties from altprof
              { exclude_filters }
}}
"""
```

File: nvsvocprez/app.py [hunk 2]
```diff
@@ -1536,19 +1536,26 @@ def __init__(self, request):
...
+                        concept_profiles[alt_data["token"]] = Profile(
+                            uri=alt_data["url"],
+                            id=alt_data["token"],
+                            label=alt_data["name"],
+                            comment=alt_data["vocprezdesc"],
+                            mediatypes=["text/html"] + RDF_MEDIATYPES,
+                            default_mediatype="text/html",
+                            languages=["en"],
+                            default_language="en",
+                        )
```
Comment (roje-bodc) [inline nvsvocprez/app.py:1558]: I have a method get_alt_profile_objects() in my branch to generate the Profile objects for each alternate profile. May be able to be reused here 🤔

File: nvsvocprez/app.py [hunk 3]
```diff
@@ -1810,8 +1811,13 @@ def _render_sdo_rdf(self):
         )
         return self._render_sparql_response_rdf(sparql_construct(q, self.mediatype))
 
-    def _render_puv_rdf(self):
-        q = """
+    def _render_profile_rdf(self):
+        exclude_filters = ""
+        exclude_profiles = [alt["url"] for alt in self.alt_profiles.values() if alt["token"] != self.profile]
+        for ep in exclude_profiles:
+            exclude_filters+= f'FILTER (!STRSTARTS(STR(?p), \"{ep}\"))\n'
```
Comment (roje-bodc) [inline nvsvocprez/app.py:1818]: Are the escaped quotes here needed? I think un-escaped should be fine as you've wrapped the string in single quotes.

**Review by roje-bodc — COMMENTED** [review]
Looks good, just one comment about quote escaping.

File: nvsvocprez/app.py (hunk 3 above)
Comment (matcor-bodc) [reply nvsvocprez/app.py:1818]: Yep, that's a relic from when there was alt["url"] instead of ep

**Pull request merged**